config/initializers/ahoy.rb

```diff
@@ -3,6 +3,25 @@
 Ahoy.api_only = true
 
 module Ahoy
-  class Store < Ahoy::Stores::LogStore
+  class Store < Ahoy::Stores::ActiveRecordTokenStore
+  end
+
+  module Stores
+    class ActiveRecordTokenStore < BaseStore
+      def track_event(name, properties, _options)
+        AhoyEvent.create!(
+          user_id: user_id_from_uuid(properties),
+          name: name,
+          properties: properties
+        )
+      end
+
+      private
+
+      def user_id_from_uuid(properties)
+        user = User.find_by_uuid(properties[:user_id]) || AnonymousUser.new
```

Comment:
Should we check the `properties[:user_id]` for "anonymous" and save a DB hit potentially? Or could we just make this use the foreign key be the UUID?